modules/apps/document-library/document-library-api/src/main/java/com/liferay/document/library/preview/exception/DLPreviewSizeException.java

```diff
@@ -13,18 +14,53 @@
 public class DLPreviewSizeException extends PortalException {
 
 	public DLPreviewSizeException() {
+		this(0);
+	}
+
+	public DLPreviewSizeException(long maxSize) {
```

Comment:
Can you please call it `maxFileSize`? I can foresee the future: the configuration attribute will be called this way :-D

Reply:
I will read the future and call it this way :P